lib/src/authentication/widgets/input_password.dart

```diff
@@ -64,8 +72,11 @@ class _InputPasswordState extends State<InputPassword> {
   }
 
   String? _validate(String? value, AppLocalizations l10n) {
-    if (widget.isRequired && (value == null || value.isEmpty)) {
-      return l10n.input_password_msg_missing_password;
+    if (widget.isRequired) {
+      if (value == null || value.isEmpty) {
+        return l10n.input_password_msg_missing_password;
+      }
+      return widget.validate(value);
```

Comment:
Instead of triggering it only when it's required, I would trigger it if the value isn't empty. Even when it's not required we should validate the field if something is entered

Reply:
fixed